packages/utils/plugins-helpers/src/federation.ts

```diff
@@ -275,7 +291,10 @@ export class ApolloFederation {
  * Checks if Object Type is involved in Federation. Based on `@key` directive
  * @param node Type
  */
-function isFederationObjectType(node: ObjectTypeDefinitionNode | GraphQLObjectType, schema: GraphQLSchema): boolean {
+export function checkObjectTypeFederationDetails(
```

Comment:
This function checks whether a node has is federation, and returns federation metadata for the node (for now, it returns an array of resolvable `@keys`)

I'm not very happy with the name `checkObjectTypeFederationDetails` as it reads a bit clumsy 🤔 Open for suggestion.